dotCMS/src/main/java/com/dotcms/api/system/event/SystemMessage.java

```diff
@@ -0,0 +1,47 @@
+package com.dotcms.api.system.event;
+
+import java.io.Serializable;
+import java.util.Set;
+
+/**
+ * A system message encapsulates a simple message (could be raw or rich) and a list of portlets (optional) that can apply the
+ * message.
+ * This will be the payload of a system message on:
+ * <ul>
+ *     <li>RAW_MESSAGE</li>
+ *     <li>RAW_ERROR_MESSAGE</li>
+ *     <li>RICH_MEDIA_MESSAGE</li>
```

Comment:
Here in the Doc is missing the CONFIRMATION_MESSAGE option

Reply:
The confirmation message is a nested object and it is encapsulate out side of the SystemMessage, it is called SystemConfirmationMessage